### DISCUSSION TITLE
activate extended colorwheel

### DISCUSSION THREAD

**Discussion by InfGame** [voidlabs/mosaico#690] [Q&A]
tinymce has a feature for an extended colorwheel that I would like to use inside of mosaico: https://www.tiny.cloud/docs-4x/plugins/colorpicker/
is there any way. be it in a config file or manually in code where i could activate said feature for my instance?

**Comment by bago** [comment]
Read the issue #310 and the other issues in the first reply there.

**Reply by InfGame** [reply]
Thank you! I think i have found what I needed in wysiwygs.js

```
 fullOptions: {
    toolbar1: 'bold italic forecolor backcolor hr styleselect removeformat | link unlink | pastetext code',
    //toolbar1: "bold italic | forecolor backcolor | link unlink | hr | pastetext code", // | newsletter_profile newsletter_optlink newsletter_unsubscribe newsletter_showlink";
    //toolbar2: "formatselect fontselect fontsizeselect | alignleft aligncenter alignright alignjustify | bullist numlist",
    plugins: ["link", "hr", "paste", "lists", "textcolor", "code"],
    // valid_elements: 'strong/b,em/i,*[*]',
    // extended_valid_elements: 'strong/b,em/i,*[*]',
    // Removed: image fullscreen contextmenu 
    // download custom:
    // jquery version con legacyoutput, anchor, code, importcss, link, paste, textcolor, hr, lists
  },
```
  
  I think i just need to add the colorpicker into the plugin array.

**Reply by InfGame** [reply]
nvm. that seemed to not have worked.

Mosaico or rather tiny cant seem to load or find the plugin.

**Comment by InfGame** [comment]
I have tried all the things that I have found worked for others in the issue you gave me. but with everythnig i try I just get this error back: 
`Failed to load plugin: colorpicker from url http://localhost:9006/rs/plugins/colorpicker/plugin.min.js`

any other idea on how i could get it to work? thanks for your help @bago
